crates/wash-cli/src/bin/wash.rs

```diff
@@ -236,6 +245,15 @@ async fn main() {
         .with_env_filter(EnvFilter::from_default_env())
         .init();
 
+    // Implement clap_markdown for markdown generation of command line documentation
+    let mdargs = Cli::parse();
+
+    // Most straightforward way to invoke is probably `wash app list --markdown-help > help.md`
+    if mdargs.markdown_help {
+        clap_markdown::print_help_markdown::<Cli>();
+        std::process::exit(0);
+    };
+
```

Comment:
I think, if I understand this correctly, we should move this down to line ~388 to avoid double parsing the CLI. Thoughts?

Reply:
Moved and adjusted to avoid the double-parse! Just let me know if another spot would be better.